vernac/ppvernac.ml

```diff
@@ -158,8 +158,6 @@ open Pputils
       | SearchHead c -> keyword "SearchHead" ++ spc() ++ pr_p c ++ pr_in_out_modules b
       | SearchPattern c -> keyword "SearchPattern" ++ spc() ++ pr_p c ++ pr_in_out_modules b
       | SearchRewrite c -> keyword "SearchRewrite" ++ spc() ++ pr_p c ++ pr_in_out_modules b
-      | SearchAbout sl ->
-         keyword "SearchAbout" ++ spc() ++ prlist_with_sep spc pr_search_about sl ++ pr_in_out_modules b
       | Search sl ->
          keyword "Search" ++ spc() ++ prlist_with_sep spc pr_search_about sl ++ pr_in_out_modules b
```

Comment:
Not a big deal, but since you've renamed the `search_about` and `search_about_filter` functions, you could also rename the `pr_search_about` function.